src/dp.rs

```diff
@@ -19,50 +63,49 @@ pub trait DifferentialPrivacyBudget {}
 /// Marker trait for differential privacy scalar noise distributions.
 pub trait DifferentialPrivacyDistribution {}
 
-/// Zero-concentrated differential privacy (zCDP) budget as defined in [[BS16]].
+/// Zero-concentrated differential privacy (ZCDP) budget as defined in [[BS16]].
 ///
 /// [BS16]: https://arxiv.org/pdf/1605.02065.pdf
 pub struct ZeroConcentratedDifferentialPrivacyBudget {
-    /// Parameter `epsilon`, using the notation from [[CKS20]] where `rho = (epsilon**2)/2`
-    /// for a `rho`-zCDP budget.
+    epsilon: Ratio<BigUint>,
+}
+
+/// Alias for ZeroConcentratedDifferentialPrivacyBudget.
+pub type ZCdpBudget = ZeroConcentratedDifferentialPrivacyBudget;
+
+impl ZCdpBudget {
+    /// Create a budget for parameter `epsilon`, using the notation from [[CKS20]] where `rho = (epsilon**2)/2`
+    /// for a `rho`-ZCDP budget.
     ///
     /// [CKS20]: https://arxiv.org/pdf/2004.00010.pdf
-    pub epsilon: Rational,
+    pub fn new(epsilon: Rational) -> Self {
+        ZCdpBudget { epsilon: epsilon.0 }
```

Comment:
```suggestion
        Self { epsilon: epsilon.0 }
```

Reply:
Done.

Out of curiosity: why do you prefer to use `Self` here, while [above](https://github.com/divviup/libprio-rs/pull/578#discussion_r1267316321) you prefer the normalized version?

Reply:
That's a reasonable question if I've heard of. Partially because this is consistent with feedback I've gotten from @tgeoghegan, who basically taught me rust. I also think it's relatively clear what the `Self` is, but you might have to do some digging to find out what `Self::AssociatedType` is.